deploy-board/deploy_board/templates/clusters/cluster_capacity.tmpl

```diff
@@ -66,6 +69,8 @@
                 remainingCapacity: info.placements != null ?  info.placements.reduce((s, e) => s + e.capacity, 0) : Infinity,
                 placements: info.placements,
                 originalCapacity: cluster.capacity,
+                terminationLimit: terminationLimit,
+                terminatingHostCount: terminatingHostCount,
```

Comment:
Just realized this could cause issues. I think we should check right before the request is made. Otherwise, a staled browser tab can simply bypass this.

Reply:
nope, I tried on dev1, didn't see any issue

Reply:
I mean this scenario:
1. user opens the capacity page in one tab
2. user terminate a host using a different tab
3. user come back to the first tab and the user will be able to change capacity because `terminatingHostCount==0`